contracts/lib/eigenlayer-middleware/lib/ds-test/src/test.sol": No such file or directory

An attempt to exec `make build-contracts` fails with the following error.

```console
$ make build-contracts
cd contracts && forge build
[⠃] Compiling...2024-07-01T14:49:01.902488Z ERROR foundry_compilers_artifacts_solc::sources: error="/home/khassanov/Workspace/github.com/Layr-Labs/incredible-squaring-avs/contracts/lib/eigenlayer-middleware/lib/ds-test/src/test.sol": No such file or directory (os error 2)
[⠊] Compiling...
Error: 
failed to resolve file: "/home/khassanov/Workspace/github.com/Layr-Labs/incredible-squaring-avs/contracts/lib/eigenlayer-middleware/lib/ds-test/src/test.sol": No such file or directory (os error 2); check configured remappings
        --> /home/khassanov/Workspace/github.com/Layr-Labs/incredible-squaring-avs/contracts/lib/forge-std/src/Test.sol
        ds-test/test.sol
make: *** [Makefile:21: build-contracts] Error 1
```

--- a/contracts/foundry.toml
+++ b/contracts/foundry.toml
@@ -11,7 +11,8 @@ remappings = [
     "@credible-squaring/=src/",
     "@openzeppelin/=lib/eigenlayer-middleware/lib/eigenlayer-contracts/lib/openzeppelin-contracts/",
     "@openzeppelin-upgrades/=lib/eigenlayer-middleware/lib/eigenlayer-contracts/lib/openzeppelin-contracts-upgradeable/",
-    "forge-std/=lib/forge-std/src/"
+    "forge-std/=lib/forge-std/src/",
+     "ds-test/=lib/forge-std/lib/ds-test/src/"
 ]
 
 gas_reports = ["*"]

--- a/contracts/foundry.toml
+++ b/contracts/foundry.toml
@@ -12,7 +12,7 @@ remappings = [
     "@openzeppelin/=lib/eigenlayer-middleware/lib/eigenlayer-contracts/lib/openzeppelin-contracts/",
     "@openzeppelin-upgrades/=lib/eigenlayer-middleware/lib/eigenlayer-contracts/lib/openzeppelin-contracts-upgradeable/",
     "forge-std/=lib/forge-std/src/",
-     "ds-test/=lib/forge-std/lib/ds-test/src/"
+    "ds-test/=lib/forge-std/lib/ds-test/src/"
 ]
 
 gas_reports = ["*"]

